twilio/task_router/taskrouter_config.py

```diff
@@ -10,8 +10,31 @@ class TaskRouterConfig:
     """
 
     def __init__(self, rules, default_target):
-        self.filters = rules
+
         self.default_filter = default_target
+        workflow_rules = []
+
+        for rule in rules:
+            if isinstance(rule, WorkflowRule):
+                workflow_rules.append(rule)
+            else:
+                try:
+                    name = rule['friendly_name']
+                except KeyError:
+                    name = rule['filter_friendly_name']
+                workflow_rules.append(
+                    WorkflowRule(rule['expression'], rule['targets'], name))
+        self.filters = workflow_rules
+
+    def to_json(self):
```

Comment:
previous, how did the user convert this to json? did they have to use json.dumps?